### ISSUE TITLE
Link icon not transparent

### ISSUE THREAD

**Issue by zsviczian** [excalidraw/excalidraw#4767]
![image](https://user-images.githubusercontent.com/14358394/153278243-b505e269-c5f1-40cc-b09c-1989e430e77a.png)

**Comment by dwelle** [comment]
By design. The image above actually isn't the best argument for making it transparent :p

/cc @ad1992

**Comment by zsviczian** [comment]
I have to disagree.  One of the key features of Excalidraw is the ease at which you can draw beautiful designs. These link icons undermine that beautiful look and feel...

**Comment by dwelle** [comment]
> One of the key features of Excalidraw is the ease at which you can draw beautiful designs. These link icons undermine that beautiful look and feel...

I don't disagree, but drawing beautiful diagrams relate to the output (export), not the UI. But UI is of course important as well.

But, the purpose of the link icon is for it to be visible — an indication that an element is hyperlinked. In the screenshot above, it's almost invisible, thus goes counter to that objective.

I think the argument of this and related issues you've created is: diagrams that are very heavily hyperlinked are running into UI/UX issues with the link icon. We can discuss that in https://github.com/excalidraw/excalidraw/issues/4761#issuecomment-1034195123
